src/main/java/core/basesyntax/service/impl/ReportServiceImpl.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.service.ReportService;
+
+public class ReportServiceImpl implements ReportService {
+    @Override
+    public String createReport(Storage storage) {

```

Comment:
we can access storage directly
```suggestion
    public String createReport() {

```